__tests__/__snapshots__/test.js.snap

```diff
@@ -1540,7 +1540,7 @@ myFoo.foo(42);
 \`\`\`
 
 \`\`\`html
-<p data-foo>Data-Foo Element in the dom</p>
```

Comment:
New version of Highlight.js detect as CSS but it is wrong. I add `bar` value to better detection